Library/Homebrew/os/mac/sdk.rb

```diff
@@ -93,11 +84,15 @@ def sdk_paths
 
             # Use unversioned SDK path on Big Sur to avoid issues such as:
             # https://github.com/Homebrew/homebrew-core/issues/67075
+            # This creates an entry in `paths` whose key is the OS major version
             sdk_path = File.join(sdk_prefix, "MacOSX.sdk")
```

Comment:
```suggestion
            sdk_path = Pathname("#{sdk_prefix}/MacOSX.sdk")
```

means you can use `sdk_path.directory?`, `sdk_settings = sdk_path/"SDKSettings.json`, `sdk_settings.read`, `sdk_settings.exist?` below which makes this a bit easier to read.

Reply:
Thanks! I was wondering how to give `sdk_settings` the `read` method. I think this should be a lot easier to read now.